schemas/author.json

```diff
@@ -85,13 +98,25 @@
 			"type": "string",
 			"description": "An optional URL to this contributor's webpage.",
 			"validation": [
-				"url"
+				"url",
+				"nullable"
 			]
 		},
 		"userGroupId": {
 			"type": "integer",
 			"description": "in which user group should this contributor be identified. Usually one of the author roles, such as Author or Translator.",
 			"apiSummary": true
+		},
+		"userGroupLabel": {
+			"type": "string",
+			"description": "A localized label identifying the contributor's role in the publication, such as 'Author' or 'Translator'.",
+			"apiSummary": true,
+			"readOnly": true
+		},
+		"fullName": {
+			"type": "string",
+			"description": "The full name of the author",
```

Comment:
> The full name of the author. This will be the `preferredPublicName` or, if that is not available, a string containing the `givenName` and `familyName`.

Reply:
changed